src/main/cli/process.ts

```diff
@@ -0,0 +1,111 @@
+
+// Logger
+const debug = debug_("readium-desktop:cli");
+
+export function cli(mainFct: () => void) {
+
+    yargs
+        .scriptName(__APP_NAME__)
+        .usage("$0 <cmd> [args]")
+        .command("opds <title> <url>",
+            "import opds feed",
+            (y) =>
+                y.positional("title", {
+                    describe: "title opds feed",
+                    type: "string",
+                })
+                .positional("url", {
+                    describe: "url opds feed",
+                    type: "string",
+                })
+            ,
+            (argv) => {
+                const promise = cliOpds(argv.title, argv.url);
+                promise.then((isValid) => {
+                    if (isValid) {
+                        app.exit(0);
+                        return ;
+                    }
+                    app.exit(1);
+                }).catch(() => {
+                    app.exit(1);
+                });
+            },
+        )
+        .command("import <path>",
+            "import epub or lpcl file",
+            (y) =>
+                y.positional("path", {
+                    describe: "path of your publication",
+                    type: "string",
+                    coerce: (arg) => path.resolve(arg),
+                })
+            ,
+            (argv) => {
+                const promise = cliImport(argv.path);
+                promise.then((isValid) => {
+                    if (isValid) {
+                        app.exit(0);
+                        return ;
+                    }
+                    app.exit(1);
+                }).catch(() => {
+                    app.exit(1);
+                });
+            },
+        )
+        .command("read <title>",
+            "searches already-imported publications with the provided TITLE, and opens the reader with the first match",
+            (y) =>
+                y.positional("title", {
+                    describe: "title of your publication",
+                    type: "string",
+                })
+            ,
+            (argv) => {
+                mainFct();
+                app.on("will-finish-launching", async () => {
+                    await cliRead(argv.title);
+                });
+            },
+        )
+        .command("$0 [path]",
+            "default command",
+            (y) =>
+                y.positional("path", {
+                    describe: "publication in absolute or relative path",
+                    type: "string",
+                    coerce: (arg) => path.resolve(arg),
+                })
+            ,
+            (argv) => {
+                // const filePathArray = argv._.map((p) => path.resolve(p));
+
+                mainFct();
+                if (argv.path) {
+                    app.on("will-finish-launching", async () => {
+                        if (!await cli_(argv.path)) {
+                            debug("no publication to open");
```

Comment:
If I am not mistaken, this debug message is incorrect. It should say something like "import failed for at least one of the given publication paths" (regardless of whether the first successfully-imported publication was opened in the reader view)

Reply:
I will say "import and read failed for at least one of the given publication paths"